### PR TITLE
fix drainer initialCommitTs issue

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

`initialCommitTs` should be a `string`, itherwise failed,  for example: `initialCommitTs: 417774109500375041` will be parsed as:

image

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
Fix drainer values.yaml `initialCommitTs` bug
```

### CODE REVIEW

File: charts/tidb-drainer/values.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ storage: 10Gi
 # disbale detect causality
 disableDetect: false
 # if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint
-initialCommitTs: 0
+initialCommitTs: "0"
```
Comment (DanielZhangQD) [inline charts/tidb-drainer/values.yaml:31]: From the comment of drainer dev, operator should not set the default value or set it to "-1", which is the default value in drainer, so we can set it to `# initialCommitTs: "-1"`?
BTW, the `_start_drainer.sh.tpl` also needs to update?

Comment (weekface) [reply charts/tidb-drainer/values.yaml:31]: _start_drainer.sh.tpl don't need to update.

Comment (weekface) [inline charts/tidb-drainer/values.yaml:31]:
```suggestion
initialCommitTs: "-1"
```

Do you means change it to "-1"? @DanielZhangQD

Comment (Yisaer) [reply charts/tidb-drainer/values.yaml:31]: actuallly

Comment (DanielZhangQD) [reply charts/tidb-drainer/values.yaml]: We'd better set the default value in `_start_drainer.sh.tpl` to `-1`.

**Comment by DanielZhangQD** [comment]
@cofyc PTAL.

**Pull request merged**